indicators/views/views_program.py

```diff
@@ -285,26 +288,197 @@ def programs_rollup_export_csv(request):
+        uom_type_options = ['Number (#)', 'Percentage (%)']
+        uom_type_validation = DataValidation(
+            type="list", formula1=f'"{",".join(uom_type_options)}"', allow_blank=False)
+        validation_map["Number (#) or percentage (%)"] = uom_type_validation
+
+        dir_change_options = ['N/A', 'Increase (+)', 'Decrease (-)']
+        dir_change_validation = DataValidation(
+            type="list", formula1=f'"{",".join(dir_change_options)}"', allow_blank=True)
+        validation_map["Direction of change"] = dir_change_validation
+
+        target_freq_options = [str(freq[1]) for freq in Indicator.TARGET_FREQUENCIES]
+        target_freq_validation = DataValidation(
+            type="list", formula1=f'"{",".join(target_freq_options)}"', allow_blank=False)
+        validation_map["Target frequency"] = target_freq_validation
+
+
+        for dv in [uom_type_validation]:
+            dv.error = 'Your entry is not in the list'
+            dv.errorTitle = 'Invalid Entry'
+            dv.prompt = 'Please select from the list'
+            dv.promptTitle = 'List Selection'
+
 
         wb = openpyxl.load_workbook(filename=f'{settings.SITE_ROOT}/indicators/BulkImportTemplate.xlsx')
+        ws = wb.worksheets[0]
+        wb.add_named_style(self.title_style)
+        wb.add_named_style(self.level_header_style)
+        wb.add_named_style(self.protected_indicator_style)
+        wb.add_named_style(self.unprotected_indicator_style)
+        wb.add_named_style(self.optional_header_style)
+        wb.add_named_style(self.required_header_style)
+        ws.add_data_validation(uom_type_validation)
+        ws.add_data_validation(dir_change_validation)
+        ws.add_data_validation(target_freq_validation)
+
+        first_used_column = 2
+        last_used_column = first_used_column + len(self.COLUMNS) - 1
+        # Translators: Heading placed in the cell of a spreadsheet that allows users to upload Indicators in bulk
+        ws.cell(1, first_used_column).value = gettext("Import indicators")
+        ws.cell(2, first_used_column).value = program.name
+        ws.cell(2, first_used_column).style = 'title_style'
+        # Translators: Instructions provided as part of an Excel template that allows users to upload Indicators
+        ws.cell(4, first_used_column).value = gettext(
+            "INSTRUCTIONS\n"
+            "1. Indicator rows are provided for each result level. You can delete indicator rows you do not need. You can also leave them empty and they will be ignored.\n"
+            "2. Required columns are highlighted with a dark  background and an asterisk (*) in the header row. You can delete columns you don't need, as long as they are not required.\n"
+            "3. When youare done, upload the template to the results framework or program page."
+        )
+        # ws.merge_cells(
+        #     start_row=4,
+        #     start_column=first_used_column,
+        #     end_row=4,
+        #     end_column=10)
```

Comment:
Did you mean to leave this commented out?

Reply:
I'll just delete it.  I don't think it's needed since the initial Excel template has those cells merged.  I'll just add it back if it's needed at some point.